language_service/src/completion/text_edits.rs

```diff
@@ -27,8 +27,10 @@ impl TextEditRange {
         };
         finder.visit_package(&compilation.user_unit().ast.package);
 
-        let insert_open_at = match compilation.kind {
-            CompilationKind::OpenProject { .. } => finder.start_of_namespace,
+        let insert_open_at = match &compilation.kind {
+            CompilationKind::OpenProject { .. } | CompilationKind::OpenQASM { .. } => {
```

Comment:
What happens if we make the `OpenQASM` arm unreachable? This whole file is _so_ inapplicable to QASM, I'll be sad if we're calling this somehow in QASM cases.